From 41bc6618376cb2a87b154612e0b132761d4f4bd3 Mon Sep 17 00:00:00 2001
From: Isaac Freund <mail@isaacfreund.com>
Date: Tue, 10 Dec 2024 13:48:11 +0100
Subject: [PATCH] Fix bash completion installation

Currently this fails if the directory for the completion does not
already exist, at least with FreeBSD install(1).
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index e298e157fbfed5bc4b1196a3bdba063c8ba07414..915786959ac59bb1a28a5b2c1d850e2a635fdb1f 100644
--- a/Makefile
+++ b/Makefile
@@ -26,6 +26,7 @@ install: wlopm
 	install wlopm $(DESTDIR)$(BINDIR)
 	install -d $(DESTDIR)$(MANDIR)/man1
 	install -m 644 wlopm.1 $(DESTDIR)$(MANDIR)/man1
+	install -d $(DESTDIR)$(BASHCOMPDIR)
 	install bash-completion $(DESTDIR)$(BASHCOMPDIR)/wlopm
 
 uninstall:
